### PR TITLE
Nystromformer ONNX export

### PR DESCRIPTION
**Pull request by whr778**
This PR implements ONNX export functionality for Nystromformer models.

Fixes # ([727](https://github.com/huggingface/optimum/issues/727))

In addition to running the test cases, I exported a locally built Nystromformer 2048 input sequence model to ONNX and ran prediction. I verified the prediction output.

I decided to use the uw-madison/nystromformer-1024 for testing and not the hf-internal-testing/tiny-random-NystromformerModel... as the inputs are wrong in the hf-internal-testing model.

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you make sure to update the documentation with your changes?
- [X] Did you write any new necessary tests?

### CODE REVIEW

File: docs/source/exporters/onnx/package_reference/configuration.mdx [hunk 1]
```diff
@@ -107,6 +107,7 @@ They specify which input generators should be used for the dummy inputs, but rem
 - MobileVit
 - MPNet
 - OwlVit
+- Nystromformer
```
Comment (fxmarty) [inline docs/source/exporters/onnx/package_reference/configuration.mdx:110]:
```suggestion
- Nystromformer
- OwlVit
```

File: optimum/onnxruntime/utils.py [hunk 2]
```diff
@@ -93,6 +93,7 @@ class ORTConfigManager:
         "mbart": "bart",
         "mt5": "bart",
         "m2m_100": "bart",
+        "Nystromformer": "bert",
```
Comment (fxmarty) [inline optimum/onnxruntime/utils.py:96]:
```suggestion
        "nystromformer": "bert",
```

**Review by fxmarty — APPROVED** [review]
Thank you for your contribution, very clean! Just to understand, what is the issue with `hf-internal-testing/tiny-random-NystromformerModel` with the ONNX Runtime integration?

**Comment by fxmarty** [comment]
I tested locally with `"nystromformer": "hf-internal-testing/tiny-random-NystromformerModel"` and `pytest tests/onnxruntime/test_modeling.py -k "nystromformer" --exitfirst -s` and the tests pass.

Could you have a try again with the tiny model? It avoids to slow down too much the tests.

**Comment by whr778** [comment]
Sure.  I'll do that right now and push a change if it passes.

**Comment by whr778** [comment]
I cleaned out all of my pycache files etc ```find . | grep -E
"(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf``` and the test ran fine.  I'll
push a change and notify you when it's done.

**Comment by whr778** [comment]
@fxmarty changes are pushed.  Thanks for the help!

**Comment by fxmarty** [comment]
Thanks great! The git history seems broken (see the diff), could you try to fix it? Not sure what's wrong, I already had the issue using git rebase.

**Comment by whr778** [comment]
@fxmarty I think I have the git history fixed. I also ran make style.  What appears to have happened is I inadvertently hit the sync fork button on the forked optimum repository web page.
I fixed it with a git merge upstream/main. E.g.
```
 1039  make style
 1040  git status
 1041  history | grep git
 1042  git fetch upstream
 1043  git merge upstream/main
 1044  git status
 1045  git push -u origin onnx-export-nystromformer
```
Sorry about the issues.

**Comment by fxmarty** [comment]
It's all good now, the failing test is unrelated. Thanks for the addition!

**Pull request merged**